app/assets/javascripts/bastion/bastion.module.js

```diff
@@ -159,17 +159,19 @@ angular.module('Bastion').run(['$rootScope', '$state', '$stateParams', 'gettextC
 
         // Set the current language
         gettextCatalog.currentLanguage = currentLocale;
-
         $rootScope.$on('$stateChangeStart',
             function () {
-            //save location.search so we can add it back after transition is done
-            this.locationSearch = $location.search();
-        });
+                //save location.search so we can add it back after transition is done
+                this.locationSearch = $location.search().search;
+            });
```

Comment:
Ahh, the last line is not typically indented like (at least looking at other examples within the code).